### ISSUE TITLE
refreshToken not working

### ISSUE THREAD

**Issue by GSFZamai** [Haskkor/react-native-recaptchav3#25]
Hi there!
I noticed that the call of the refreshToken on the reference stopped working on my app.
For now as work around I'm using React state to render and hide the Recaptcha component.
Anyone else having the same issue?

**Comment by Sewb21** [comment]
Same here

**Comment by GSFZamai** [comment]
Hi there, i did some tests here. And seems to be related to the react-native-webview.
In my case, when the react-native-webview is updated to 11.20, some way it conflicts with the recaptcha's webview dependence and refreshToken stops working. When I downgrade to 11.18, it works perfectly.

**Comment by Sewb21** [comment]
Hey, so I thought this might be the case so I downgraded react-native-webview but I am using Expo on the latest SDK 46 and I encountered some build issues. I've reverted back to Expo 45 anyway for the time being due to other reasons

**Comment by GSFZamai** [comment]
But I did one workaround (before I downgrade webview).
Created one state to show/hide my Recaptcha component, and when I need to refresh token, I set to true, after processing the token, on the end of onReceiveToken method just set to false.
Every time you set true, it'll re-render and generete a new token, proccess and destroy your component.

**Comment by Sewb21** [comment]
Yeah, I managed to get something like this working so I guess will have to be the solution until/if the package gets fixed. Thank you @GSFZamai 😄

**Comment by amitkumarcoding** [comment]
Use this 

https://www.npmjs.com/package/react-native-secure-captcha-v3
